packages/backend/src/ee/models/AiAgentModel.ts

```diff
@@ -286,7 +285,8 @@ export class AiAgentModel {
             if (args.instruction) {
                 await trx(AiAgentInstructionVersionsTableName).insert({
                     ai_agent_uuid: agent.ai_agent_uuid,
-                    created_at: createdAt,
+                    // @ts-expect-error - knex types are not correct
+                    created_at: trx.fn.now(),
```

Comment:
I think we should update the Db types that use Date to also be `Knex.Raw`
so we won't need to bypass ts
something like:

```ts
type DatabaseTimestamp = Date | Knex.Raw;

export type DbAiAgentInstructionVersions = {
    ai_agent_instruction_version_uuid: string;
    ai_agent_uuid: string;
    instruction: string;
    created_at: DatabaseTimestamp;
};
```

wdyt?